src/NetAppFiles/NetAppFiles/help/Az.NetAppFiles.md

```diff
@@ -0,0 +1,55 @@
+---
+Module Name: az.NetAppFiles
```

Comment:
@leonardbf just to be safe, I would make sure the first character is capitalized